src/System.Buffers.Primitives/System/Buffers/OwnedBuffer.cs

```diff
@@ -69,7 +40,11 @@ public void Dispose()
         public abstract void Retain();
 
         public abstract void Release();
-
         #endregion
+
+        protected static unsafe void* Add(void* pointer, int offset)
```

Comment:
Might not be exactly that since I'm going from memory. But then you don't have to do the sizeof math and it all gets inlined anyways.